canvas_modules/common-canvas/src/command-actions/deleteObjectsAction.js

```diff
@@ -172,16 +172,29 @@ export default class DeleteObjectsAction extends Action {
 	}
 
 	createActionLabel() {
+
+		const stringList = [
+			{ label: "Nodes", val: this.nodesToDelete.length + this.supernodesToDelete.length },
+			{ label: "Comments", val: this.commentsToDelete.length },
+			{ label: "Links", val: this.linksToDelete.length }
+		];
+		const dynamicStringKey = "action.delete" + stringList
+			.filter((v) => v.val).map((o) => o.label)
+			.join("");
+
 		if (this.areDetachableLinksSupported) {
-			return this.labelUtil.getActionLabel(this, "action.deleteNodesCommentsLinks",
+			return this.labelUtil.getActionLabel(this, "action.delete" + dynamicStringKey,
```

Comment:
This cannot work because you've already added "action.delete" at the beginning of dynamicStringKey. This suggests you haven't tested this part of the code, which means you've asking me to review untested code. 

If you're not sure how to get the code you've changed to run in a test scenario then please ask. In this case you need to switch the `enableLinkSelection` config variable to "Detachable".